lib/voice/VoiceConnection.js

```diff
@@ -78,13 +78,22 @@ class VoiceConnection extends EventEmitter {
      */
     connecting = false;
     connectionTimeout = null;
+    /**
+     * The amount of consecutive failures encountered when decrypting.
+     */
```

Comment:
Is this supposed to be publicly facing? If so:
```suggestion
    /**
     * The amount of consecutive failures encountered when decrypting.
     * @type {Number}
     */
```

and add TS defs appropriately